llama_stack/providers/adapters/safety/bedrock/bedrock.py

```diff
@@ -104,10 +135,14 @@ async def run_shield(
                 # guardrails returns a list - however for this implementation we will leverage the last values
                 metadata = dict(assessment)
 
-            return SafetyViolation(
-                user_message=user_message,
-                violation_level=ViolationLevel.ERROR,
-                metadata=metadata,
+            return RunShieldResponse(
+                violations=[
```

Comment:
there are multiple violations? I don't think we ever defined multiple violations if IIRC

Reply:
true, this should be just one. changing.